saleor/graphql/product/mutations/attributes.py

```diff
@@ -317,6 +317,18 @@ def get_operations(cls, info, operations: List[AttributeAssignInput]):
             pk = from_global_id_strict_type(
                 operation.id, only_type=Attribute, field="operations"
             )
+
+            try:
+                models.Attribute.objects.get(pk=pk)
```

Comment:
It's not an optimal solution - it performs a database query for each operation. You could instead do one query always if you used `Attribute.objects.filter(id__in=all_attribute_ids)` or something like that and compare the number of instances you got from DB with the number of IDs passed in the input. 

There is also `clean_operations` function which does other validations as well, so this one would fit there.

We are able to pass additional context in the error response to tell the user which IDs are wrong, @IKarbowiak did that in one of her PRs; Iga can you paste some quick example how can we pass attribute IDs to the error context?

Reply:
sure I'll fix fix that

Reply:
1) Extend `ErrorClass` for the additional field with attributes ids
2) To add ids to `ValidationError` you need to provide it to `params` attribute in form of dict, where `key` is name of field from error class, and `value` is list of ids or something else. Like that:
`ValidationError(message=msg, code=code, params={"field_name": [IDs]})`
3) You must also update handle_errors. We have a solution for that in BaseMutation but it's still on my branch, unfortunately 😞. So for now you can just override `handle_typed_errors` as is done here https://github.com/mirumee/saleor/pull/5399/files#diff-5cc938d1fb7222180f0de3a3eb4fb4e1R102 .

Reply:
@IKarbowiak something like this?